examples/Layout/GridWrapper.md

```diff
@@ -0,0 +1,30 @@
+*Note: this example is a placeholder until the `GridContainer` and `GridItem` components are finished.
+
+The `GridWrapper` is the top-level of the grid structure. It defines the 12-column grid, with space between rows and columns. It can be used to embed grids within other components, letting us define sub-grid arrangements when needed.
+
+It does not apply any margins, padding, additional borders shadows, or other styling, which should be handled by parent/child components instead.
+
+```jsx
+    <div>
+    <h2>Small Gap (default)</h2>
+      <GridWrapper gap="small">
```

Comment:
Since this is meant to show the default, maybe we can leave out the `gap="small"`? Or break it into two examples showing "no value" and "small"?

Reply:
Forgot to change this and the previous issue after moving some things around, thanks for catching these. I'm planning to make a more robust example (like the Table example we talked about) after I finish all 3 components, so this is mostly just a placeholder so I can see how the component is functioning as I'm working on it.